pandas/tests/tseries/offsets/test_offsets.py

```diff
@@ -92,7 +93,7 @@ def test_to_M8():
 
 
 class Base:
-    _offset = None
+    _offset = None  # type: Union[BusinessHour, CustomBusinessHour, Callable]
```

Comment:
Can these all just be replaced with `DateOffset`? Looks like they all might inherit from that

Reply:
BusinessHour, CustomBusinessHour can be replaced by `DateOffset`. Unfortunately, some of the subclasses of base use the `_offset` as an alias for function, hence the reason for `Callable`.

Reply:
use `Type[DateOffset]`